### DISCUSSION TITLE
Brew formula for MacOS auditing looks buggy or weird

### DISCUSSION THREAD

**Discussion by AkechiShiro** [Homebrew/discussions#3523] [Tap maintenance and Homebrew development] [answered]
Hi,

I'm packaging a software that depends on ruby@3.1.

The command `brew audit --strict --new-formula --online nameOfFormula`

Tells me the following warnings :
- Dependency "ruby@3.1" is provided by macOS; please replace 'depends_on' by 'uses_from_macos'

So I did that and then ran again the audit and was greeted by this warning :
- col 3: `uses_from_macos` should only be used for macOS dependencies, not ruby@3.1 

So I figured I'd put both `depends_on` AND `uses_from_macos` in my formulae, things is I get both warning now.

Is this a bug or an intended warning that I'm misunderstanding ?

If needed I can provide the source formulae.

**Comment by gromgit** [comment, marked as answer]
This _might_ be a bug, as `ruby@3.1` is just an alias for `ruby`.

Try `uses_from_macos "ruby"` for now.
